client/connection.go

```diff
@@ -45,6 +45,9 @@ type ConnectionArgs struct {
 
 	// Cookie jar
 	CookieJar http.CookieJar
+
+	// Skip automatic GetServer request upon connection
+	SkipGetServer bool
```

Comment:
Yeah I know, but I don't have many other solutions since the "lxd shutdown" command uses lxd.ConnectLXDUnix to send a raw request to /internal/shutdown. Now, if that would involve a database query (like /api/1.0 does), you would not be able to shutdown the last node of a cluster (since it has lost leadership/quorum).

Reply:
I guess I could think of a few tricks to avoid exposing this in ConnectionArgs or interfaces.go, if that's desired. Or we could just add a comment that this is for internal LXD usage, ISTR there are other client interfaces marked as such.

Reply:
Hmm so isn't there a bigger problem with the shutdown logic when clustered then?
Calling `lxd shutdown` will trigger a clean shutdown of all containers, which certainly involves database access.

Reply:
For shutdown, lets do this:
 - Keep a list of running containers in memory, something like:
```go
type runningContainer struct {
    Name string
    StopPriority int
    HostShutdownTimeout int
}
```
 - Modify containersShutdown to use that struct
 - For each of those, we need to load them through go-lxc lxc.NewContainer
 - Shutdown and final Stop should then be handled through go-lxc in the clustered case, eliminating all DB calls
 - Modify internalContainerOnStop to know that we're shutting down and that the database isn't available, in such case, it's fine for it to be a no-op

Reply:
As agreed during the catch-up, this is not blocking and will be done after this initial PR is merged.